packages/manager/modules/pci/src/projects/project/storages/containers/container/container.controller.js

```diff
@@ -0,0 +1,136 @@
+import get from 'lodash/get';
+
+import { CONTAINER_DEFAULT_USER } from '../containers.constants';
+
+export default class PciStoragesContainersContainerController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $rootScope,
+    $translate,
+    $window,
+    CucCloudMessage,
+    PciProjectStorageContainersService,
+  ) {
+    this.$q = $q;
+    this.$rootScope = $rootScope;
+    this.$translate = $translate;
+    this.$window = $window;
+    this.CucCloudMessage = CucCloudMessage;
+    this.PciProjectStorageContainersService = PciProjectStorageContainersService;
+
+    this.defaultUser = CONTAINER_DEFAULT_USER;
+  }
+
+  $onInit() {
+    this.$rootScope.$on('pci_storages_containers_container_refresh', () => this.refreshContainer());
+
+    this.columnsParameters = [{
+      name: 'retrievalState',
+      hidden: !this.archive,
+    }];
+
+    this.containers = null;
+
+    this.loading = true;
```

Comment:
Since you are dealing with only one loader then I encourage you to prefer using `isLoading`.